_commands/tools/curl.md

```diff
@@ -2,22 +2,63 @@
 ---
 curl
 -------
-TODO: Add documentation for this command by submitting a pull request.
+`curl` transfers data to/from a server, using one of the supported protocols. The supported protocols include DICT, FILE, FTP, FTPS, GOPHER, HTTP, HTTPS, IMAP, IMAPS, LDAP, LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMB, SMBS, SMTP, SMTPS, TELNET and TFTP.
 <!-- one line explanation would go here -->
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ curl http://www.google.com
 ~~~
+This will output the content of of the url to the command line
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
+#### `curl -o filename url`
 
-##### Break it down
+Saves the curl command to the given filename
```

Comment:
Do you mean saves the output? If so, what is the output? Is it going to be a webpage, text, or something different?